webapp/src/components/ImageFieldEditor.tsx

```diff
@@ -224,6 +224,7 @@ export class ImageFieldEditor<U extends pxt.Asset> extends React.Component<Image
             );
         }
 
+        type = this.state.editingTile ? pxt.AssetType.Tile : type;
```

Comment:
is the wrong asset type being passed into this function?

Reply:
oh it thinks it's still in the Tilemap editor so it's looking at the wrong asset

Reply:
ah! hmm maybe this could be passed in in the this.filterAssets(this.userAssets) call like it is for gallery assets? or you could remove it from the gallery assets call and just use this

Reply:
oh yeah that makes sense! I'll do that. thanks shannon :D